packages/examples/main/src/browser/main.ts

```diff
@@ -121,7 +121,7 @@ const createEditor = async () => {
         validate();
     });
 
-    workspace.onDidOpenTextDocument((_event) => {
+    workspace.onDidChangeTextDocument((_event) => {
```

Comment:
was it done on purpose?

Reply:
This was unstable. Sometime it worked. Sometime it didn't. I already forgot I changed this last night. Should we investigate?

Reply:
isn't it a race condition? you should probably search for your file inside workspace.textDocuments

Reply:
ok, there seems to be a problem `onDidOpenTextDocument` is really never called. I added log statements locally and I don't see it. Will try in other examples. Maybe it broke earlier. The example still work, but you see errors in the log.

Reply:
My fault, it is the order. The callback `onDidOpenTextDocument`. is added to late. I will fix this

Reply:
Done ⬇️